File: Leisure Land/Backend/server/model/reservation.js
const mongoose = require('mongoose');
const Schema= mongoose.Schema;



const reservation = new Schema ({
    username: {
        type: String,
       
        
    },
    title: {
        type: String,
        
    },
    price: {
        type: String,
        
    },
    phone: {
        type: String,
        
    },

    date:{
        type: String,
        

    },
    

}, {collection: 'Reservations'}

);


const Reservations = mongoose.model('reservation', reservation);
module.exports = Reservations;

File: Leisure Land/leisur_diary/lib/models/reservation.dart
class Reservation {
  final String? username;
  final String? title;
  final String? date;
  final String? price;
  final String? phone;

  Reservation({this.username, this.title, this.date, this.price, this.phone});

  Map<String, dynamic> toJson() => {
        'username': username,
        'title': title,
        'date': date,
        'price': price,
        'phone': phone,
      };
}
